src/routes/blog/release-spyder-6.0.0/+page.md

```diff
@@ -0,0 +1,137 @@
+- [Juan Sebastian Bautista](https://github.com/jsbautista) developed improvements to the Windows installers, the QtConsole GUI powering our IPython Console, and Spyder's UI/UX
+- [Hendrik Louzada](https://github.com/hlouzada) created the backend for the whole remote development architecture
+- [Edgar Margffoy](https://github.com/andfoy) and [Stephannie Jimenez](https://github.com/steff456) created a brand new, pluggable and configurable Run architecture for Spyder as well as several other major API/under the hood improvements
+- [Andrés Montoya](https://github.com/conradolandia) helped improve the UI/UX, design and theming of Spyder 6 and created this brand new website for the project
+- And of course, [Carlos Cordoba](https://github.com/ccordoba12) lead the project throughout Spyder 6 development, both managing and supporting the team as well as making innumerable contributions of his own
+
+Three cheers for everyone who made Spyder 6 a reality!
+
+
+## What's next
+
+Spyder 6.0 will continue to receive point release updates with bug fixes and minor improvements, but we're already working on the next big release for Spyder, 6.1!
+Closer to a major release than a typical minor release, it'll have many further major new features and enhancements, including:
+
+- Huge improvements to the remote development platform first introduced in 6.0, including the ability to easily open, manage and run remote files just as if you were working locally.
+- A new built-in package and environment manager that'll let you create Conda environments (automatically linked to projects) and install packages right from the Spyder graphical interface, leveraging our existing Conda-based installers—no more needing to install your own Python, use a separate application or the command line to create environments and install packages, or remember to activate the right environment before installing or running.
+- A graphical Spyder plugin browser, allowing you to find, install and manage plugins with one click in standalone-installed Spyder.
+- Many other improvements to the Editor, Console, Variable Explorer, Find page, completion, Run architecture, theming and more!
+
+Want to start using Spyder 6 right now?
+The easiest way is by downloading our own installers for Windows, macOS and Linux [from our website](/download).
+You can also [install it via a variety of other methods](https://docs.spyder-ide.org/installation.html), such as  Conda and Conda distributions, Pip, WinPython, OS and distro package managers and from source on GitHub.
+
+Keep in mind that if you use our standalone installers and want to install your own Python packages not included with Spyder, until the new built-in environment manager is included in Spyder 6.1 you'll need to have another Python environment installed on your machine and select it in Spyder, if you don't have one already configured for Spyder 5.
```

Comment:
Add "for now" to imply we hope to add the capability at some point in the future, without committing to any specifics. Otherwise the same as your comment.

```suggestion
Keep in mind that if you use our standalone installers and want to install your own Python packages not included with Spyder, for now you'll need to have another Python environment installed on your machine and select it in Spyder, if you don't have one already configured.
```